articles/stream-analytics/stream-analytics-parallelization.md

```diff
@@ -71,7 +71,7 @@ The following sections discuss some example scenarios that are embarrassingly pa
 ### Simple query
 
 * Input: Event hub with 8 partitions
-* Output: Event hub with 8 partitions
+* Output: Event hub with 8 partitions ("Partition key column" must be set to use "PartitionId")
```

Comment:
do we need to specify the compat level for which it needs to be done?

Reply:
Same as above. This is required for all compat level for now.